conans/test/command/search_test.py

```diff
@@ -244,7 +244,7 @@ def _assert_pkg_q(self, query, packages_found, remote):
         if remote:
             command += " -r %s" % remote
         self.client.run(command, ignore_error=True)
-        print self.client.user_io.out
+        print(self.client.user_io.out)
```

Comment:
Actually this was  my mistake, I left a debugging print there 😅 
Together with the ``ignore_error=True`` could be removed.